Gruntfile.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(grunt) {
         ''
     ].join('\n');
 
-    var footer = '\n })();';
+    var footer = '\n })();\nmodule.exports = \'angular-svg-round-progressbar\';';
```

Comment:
This isn't the proper place to put this since it won't show up in the non-minified file. [You should put it in the module definition.](https://github.com/crisbeto/angular-svg-round-progressbar/blob/master/src/module.js)